pkg/block/mdfetcher.go

```diff
@@ -0,0 +1,103 @@
+package block
```

Comment:
`syncer`?

Same for file `mdsyncer`?

Reply:
resolved as it is related to the name discussion.